cmd/odf/ceph/ceph.go

```diff
@@ -0,0 +1,55 @@
+package ceph
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+)
+
+// CephCmd represents the ceph command
+var CephCmd = &cobra.Command{
+	Use:                "ceph",
+	Short:              "call a 'ceph' CLI command with arbitrary args",
+	DisableFlagParsing: true,
+	Hidden:             true,
```

Comment:
```suggestion
	Hidden:             true,
```
PTAL at this @travisn do we want this or not? Thanks

Reply:
Based on the offline discussion removing this.